### ISSUE TITLE
[BUG]

### ISSUE THREAD

**Issue by 91ajames** [PoloNX/SimpleModDownloader#22]
**Describe the bug**
 I was just browsing on the first 3 games that seemed to load ok until I skipped down and see why you would cheat on a puzzle game like Professor Layton.

and so it just hangs there.
Video:

**Comment by PoloNX** [comment]
It's a very weird issue. I have to test this and try to reproduce it. I think it's because in professor layton there is the char "™"

**Comment by PoloNX** [comment]
Is this the full name ?

``LAYTON’S MYSTERY JOURNEY™: Katrielle and the Millionaires’ Conspiracy - Deluxe Edition``

**Comment by 91ajames** [comment]
Yes, that is the full name. Wouldn't surprise me if that's the cause of it. Gotta love those special characters where things go take a dive and split.

**Comment by 91ajames** [comment]
Since I am here, Found another game that i was able to load whatever cheats, backing out crashed the system.
I'll edit what ever games in this comment.

**Comment by PoloNX** [comment]
There is no professor layton mods https://gamebanana.com/search?_nPage=1&_sModelName=Game&_sOrder=best_match&_sSearchString=professor

**Comment by PoloNX** [comment]
Hmm it's weird. And if you try again it works ? Maybe one issue with the download

**Comment by 91ajames** [comment]
Perhaps, not sure, finished the games I owned.

**Comment by Gamr13** [comment]
Adding to this to confirm, Pokémon Sword and Shield return no results on GameBanana (according to the app), and Legends Arceus causes a system hang, pressing the home button freezes on a grey screen. 

Atmosphere also reports a crash (and reboots) when installing a mod for Animal Crossing: New Horizons -- Specifically the Camel by Camel mod replacing one of K.K.'s songs.
